bindata/etcd/cluster-backup-pod.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cluster-backup
+  namespace: openshift-etcd
+  labels:
+    app: cluster-backup
+spec:
+  containers:
+  - name: cluster-backup
+    imagePullPolicy: IfNotPresent
+    t diff
```

Comment:
typo?

Reply:
yeah, I am reworking all of this for Monday.